test/TestExpectations.json

```diff
@@ -2504,12 +2504,6 @@
     "expectations": ["FAIL"],
     "comment": "Needs investigation, it looks like a bug in Puppeteer"
   },
-  {
-    "testIdPattern": "[network.spec] network Response.fromCache should work",
```

Comment:
Please keep an eye. I think it may start to be flaky now.

Reply:
ack. Marked as flaky for now.